app/lib/actions/remote_execution/run_host_job.rb

```diff
@@ -173,6 +194,22 @@ def determine_proxy!(proxy_selector, provider, host)
         end
         proxy
       end
+
+      extend ApipieDSL::Class
+      apipie :class, "An action representing execution of a job against a host" do
+        name 'Actions::RemoteExecution::RunHostJob'
+        refs 'Actions::RemoteExecution::RunHostJob'
+        sections only: %w[webhooks]
```

Comment:
shouldn't this be `%w[all webhooks]`?

Reply:
:facepalm: Well, yes. Probably makes sense to show everything related to `ActiveRecord` or `ActiveJob/Action` in `all` section. Although I'm not sure if an action object can be retrieved somewhere else outside of webhook payload.

Reply:
We did the same in tasks https://github.com/theforeman/foreman-tasks/pull/611/files#diff-46280f339235610e0e62cc0a085767460b6358550ef06cfd6c8a97f962572669R73 . 

As far as I know webhook payload is currently the only place where an action jail can be accessed by the users. There is a wip pr which give access to them in report templates. Does it make sense to expose those docs under `all` section if they can only be accessed in one (or two) specific context?

Reply:
Currently the `all` section holds (or at least should) all the macros/properties (except Ruby ones) even if they can be used in e.g. provisioning context only. Then there are sections to filter stuff by context.

Reply:
REX: https://github.com/theforeman/foreman_remote_execution/pull/572
tasks: https://github.com/theforeman/foreman-tasks/pull/624